droid-hal-device.inc

```diff
@@ -191,6 +197,29 @@ echo Running droid build in HABUILD_SDK
 ubu-chroot -r /srv/mer/sdks/ubu "cd %android_root; source build/envsetup.sh; lunch %{device}%{?device_variant}; rm -f .repo/local_manifests/roomservice.xml; make %{?_smp_mflags} %{?hadk_make_target}%{!?hadk_make_target:hybris-hal}"
 %endif
 
+# tools package: look first for device specific mk file, then use dhd's
+rm -f %{android_root}/system/core/mkbootimg/mkbootimg \
+      %{android_root}/system/core/libsparse/img2simg \
+      %{android_root}/system/core/libsparse/simg2img
+
+if test -f rpm/helpers/mkbootimg.mk; then
+    make -f `pwd`/rpm/helpers/mkbootimg.mk -C %{android_root}/system/core/mkbootimg/
```

Comment:
`pwd` here and below, should not be needed, as build dir will always be $ANDROID_ROOT (where rpm/ resides)